composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": "^5.6|^7.0",
```

Comment:
And probably time to bump this to `"php": ">=7.1.3",` which is the minimum for Symfony 4.4

Reply:
Updated. Thanks for review.